src/TraceEvent/Symbols/NativeSymbolModule.cs

```diff
@@ -592,6 +595,12 @@ protected override string GetSourceFromSrcServer()
                             _log.WriteLine("Source Server command {0} is not recognized as safe (sd.exe or tf.exe), failing.", fetchCmdStr);
                             return null;
                         }
+
+                        if (fetchCmdStr.Contains(OldSourceServerUrl))
```

Comment:
Nit: The .Contains check isn't really necessary. Replace returns the input string unmodified if there are no replacements.

Reply:
Good call @pharring.  Thanks.